src/main/java/carpetaddonsnotfound/mixins/ServerPlayerEntity_SpectatorPlayersUsePortalsMixin.java

```diff
@@ -39,6 +39,7 @@ Calls original move() function then performs additional action
      */
   @Override
   public void move(MovementType type, Vec3d movement) {
+    World world = this.getWorld();
```

Comment:
Vartiable world is only used once in the entire method, imo its better to inline it to that use or move it into if statement for handling nether portal.

```suggestion #1
    SyncSetInNetherPortalWithClient(this.getWorld(), pos);
```

```suggestion #2
    World world = this.getWorld();
    SyncSetInNetherPortalWithClient(world, pos);
```
Sugested code obviusly needs to be put on line 68.

Reply:
Even better I do it in the method I pass it too rather than this method

Reply:
WOW GILLY AGREED WITH ME

Reply:
Actually it is used throughout the method, hence my use of it (I missed it too reading, you'll see it's used in line 50, line 55, line 62 and line 68